docker-compose.yml

```diff
@@ -2,7 +2,8 @@
 version: '3.7'
 services:
   pmm-managed-server:
-    image: ${PMM_SERVER_IMAGE:-perconalab/pmm-server:dev-latest}
+    # REPLACE IT BEFORE MERGE    image: ${PMM_SERVER_IMAGE:-perconalab/pmm-server:dev-latest}
+    image: perconalab/pmm-server-fb:PR-1020-1a8e2cd
```

Comment:
Please use the one from https://github.com/Percona-Lab/pmm-submodules/pull/1111 when it will be there

Reply:
sure

Reply:
done